WindowManager/app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +31,98 @@
             android:configChanges="orientation|screenSize|screenLayout|smallestScreenSize|uiMode"
             android:exported="false"
             android:label="@string/window_metrics" />
+        <activity
+            android:name=".embedding.SplitActivityA"
+            android:exported="true"
+            android:label="Split Main"
+            android:configChanges="orientation|screenSize|smallestScreenSize|screenLayout|colorMode|density|touchscreen"
```

Comment:
I'm curious for the reasoning for these config change opt-outs.

When testing on the resizable emulator, it's triggering changes in display density that look like they aren't being handled.

Reply:
The idea is to avoid that a config change restart the activity, but there's no handling of the configuration changes at this stage of the sample/API.